playwright/playwright.config.ts

```diff
@@ -1,7 +1,7 @@
 import { PlaywrightTestConfig, devices } from '@playwright/test';
 
 const chromium = 'chromium' as const;
-const allSupportedBrowsers = [chromium, 'firefox', 'webkit'] as const;
+const allSupportedBrowsers = [chromium, 'firefox'] as const;
 const mobileDeviceNames = ['iPhone 11'] as const;
```

Comment:
I believe this will make Chrome give itself an iPhone user agent and screen size, but is there an Android device we can set it to so it's more logically consistent?